.env.example

```
@@ -1,369 +1,122 @@
-
-# Get from: https://console.anthropic.com/
+# =============================================================================
+# CORS
+# =============================================================================
+CORS_ORIGINS=http://localhost:3000,http://localhost:8000,https://app.devskyy.app,https://api.devskyy.app,https://staging.devskyy.com
+
+# =============================================================================
+# Frontend & API URLs
+# =============================================================================
+FRONTEND_URL=https://app.devskyy.app
+API_URL=https://api.devskyy.app
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_URL key should go before the FRONTEND_URL key

(UnorderedKey)